File: Equipos/Los_guardianes_de_los_datos/Interfaz/ordenes/Producto.py
class Producto(): 

  def __init__(self, id, nombre, receta, precio, descripcion, 
  disponibilidad, tipo, id_categoria) -> None:
    self.id = id
    self.nombre = nombre
    self.receta = receta 
    self.precio = precio 
    self.descripcion = descripcion
    self.disponibilidad = disponibilidad
    self.tipo = tipo
    self.id_categoria = id_categoria
    self.cantidad = 0

  def __str__(self) -> str:
    return self.nombre

File: Equipos/LOS INGES/WebpageProject/MiChangarro/Models/Producto.cs
using System;
using System.Collections.Generic;

namespace MiChangarro.Models
{
    public partial class Producto
    {
        public Producto()
        {
            ProductoOrdens = new HashSet<ProductoOrden>();
        }

        public int Id { get; set; }
        public string Nombre { get; set; } = null!;
        public string Descripcion { get; set; } = null!;
        public string Receta { get; set; } = null!;
        public decimal Precio { get; set; }
        public bool Disponible { get; set; }
        public int CategoriaId { get; set; }

        public virtual Categorium Categoria { get; set; } = null!;
        public virtual ICollection<ProductoOrden> ProductoOrdens { get; set; }
    }
}
